MeshLib/Properties.h

```diff
@@ -103,6 +111,14 @@ class Properties
                                                MeshItemType const item_type,
                                                int const n_components) const;
 
+    /// Non-const version of getPropertyVector returns a property vector with
+    /// given \c name, \c item_type and \c number_of_components or calls
+    /// OGS_FATAL if no such property vector exists.
+    template <typename T>
+    PropertyVector<T>* getPropertyVector(std::string const& name,
```

Comment:
for discussion: I think the `existsPropertyVector` is obsolete now, since a returned nullptr is indicating exactly the same...

Reply:
In some cases it is necessary to check the existence before accessing the property vector. Maybe using new techniques instead of exceptions, for instance, returning a tuple will be implemented in the future.